6.x/base-widgets.md

```diff
@@ -414,6 +414,64 @@ Widget Preview:
 
 <hr>
 
+<a name="livewire"></a>
+### Livewire
+
+Add a Livewire component to a page. If you haven't created your component yet, head to [Livewire documentation](https://livewire.laravel.com/docs/components) and create the component you want to use.
+
+**Important Note**: This widget automatically load the Livewire Assets (scripts and styles) for you. If you are using Livewire in the same page as this widget and is already loading the assets, you can disable the automatic loading by setting the ```livewireAssets``` attribute to ```false```.
```

Comment:
Shouldn’t we default to false? Assume people are using Livewire v3?

Reply:
that's the tricky question and why I pinged you. 

It makes sense to be false by default especially for Lw3, I just wanted to make sure we are not giving hard time for other users.

Reply:
I think that’s ok, as long as it’s documented

Reply:
I mean yes let’s default to false